community/examples/intel/README.md

```diff
@@ -436,16 +436,15 @@ See the [DFuse (DAOS FUSE)](https://docs.daos.io/v2.4/user/filesystem/?h=dfuse#d
 
 ### Delete the DAOS/Slurm Cluster infrastructure when not in use
 
-> **_NOTE:_** All data on the DAOS file system will be permanently lost after cluster deletion.
-
-> **_NOTE:_** If the Slurm controller is shut down before the auto-scale instances
-> are destroyed those instances will be left running.
+> **Note:**
```

Comment:
Pre-commit: community/examples/intel/README.md:440:3: MD032: Lists should be surrounded by blank lines (blanks-around-lists)

I think this should be like

```
> **_NOTE:_** Data on the DAOS file system will be permanently lost after cluster deletion.
> If the Slurm controller is shut down before the auto-scale instances are destroyed, those compute instances will be left running.
```

Reply:
Fixed. PyMarkdown precommit is now passing.